src/styles/layout/_header.scss

```diff
@@ -0,0 +1,215 @@
+.header {
+  background-color: $base-blue;
+  border-bottom: 6px solid #323746;
+  padding: 29px 0px 20px;
+  position: sticky;
+  top: 0;
+  z-index: 6;
+  width: 100%;
+
+  @include media-mobile {
+    border-bottom: none;
+    padding: 25px 0px 20px;
+  }
+}
+
+.header-wrapper {
+  padding: 0px 40px;
+
+  @include media-mobile {
+    padding: 0px 45px 0px 20px;
```

Comment:
I can see this small hack for centralizing
Ok for now, for future suggest to used additionally position relative + absolute